public/index.html

```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
This font is sweet! But it didn't bring in the underline that it shows on google fonts. So boo. But still cool.